### DISCUSSION TITLE
Dates are incorrect

### DISCUSSION THREAD

**Discussion by rimatech987** [gethinode/hinode#506] [Help] [answered]
**Describe the bug**

I noticed that the dates of each post are wrong.

**To reproduce**

1. git clone https://github.com/gethinode/hinode.git
2. npm install & npm run mod:update
3. npm run start:example

**Expected behavior**

That the dates are correct.

**Log file**

-

**Screenshots**

![Schermata del 2023-09-02 13-10-05](https://github.com/gethinode/hinode/assets/87300172/db8173c6-8189-4c2d-a386-266d1b1d8fc6)
![Schermata del 2023-09-02 13-09-10](https://github.com/gethinode/hinode/assets/87300172/6069b4e8-95f7-4be0-9c9d-7549d3ff9682)

**Environment (please complete the following information):**
 - Hinode version: [v0.18.7]
 - Host OS: [Fedora Linux 37 (Workstation Edition)]
 - Node version: [v18.17.1]
 - Hugo version: [hugo v0.118.2-da7983ac4b94d97d776d7c2405040de97e95c03d linux/amd64 BuildDate=2023-08-31T11:23:51Z VendorInfo=gohugoio]
 - Browser: [Mozilla Firefox 116.0.3 (64-bit) - Chromium Version 116.0.5845.140 (Official Build) (a 64 bit)]

**Additional context**

I have tried everything but to no avail (deletion of browser data, cookies, cache) but I don't understand why, for example in this post, the last modification date is shown when it does not exist within post four-post.md.

I've tried changing it, adding the last modification date of other posts, but no luck.

The creation date seems fine instead. Can you have a look?

I'm also wondering what's the point of specifying the author of the post since it doesn't appear in the header. Is there any way to show it?

**Comment by markdumay** [comment, marked as answer]
This is by design. You might want to check the [Hugo docs](https://gohugo.io/getting-started/configuration/#configure-dates). Hinode uses the creation date and last modification date (`lastmod`). Your modification date can vary with for example the date of your local file, or the git merge date.

In your example, Hugo detects that you have made changes since the original date (Jan 1st). You can either update the `date` field to the correct one, or adjust the configuration in your frontmatter.

> I'm also wondering what's the point of specifying the author of the post since it doesn't appear in the header. Is there any way to show it?

You would have to modify the desired layout page, for example `layouts/_default/single.html`. The [Hugo docs](https://gohugo.io/variables/page/#access-nested-fields-in-front-matter) provide an example on how to access the page's frontmatter.
